ansible/configs/sap-smart/sat_workload.yml

```diff
@@ -0,0 +1,126 @@
+- name: 'Install required packages'
+  yum:
+    name: '{{ item }}'
+    state: installed
+  loop:
+    - satellite
+    - firewalld
+    - python-firewall
+
+- name: 'Ensure firewalld is running'
```

Comment:
Just out of curiosity, why are you enabling and configuring firewalld here and then disabling it at the end of this task list?

Reply:
There is an easy explanation, I was doing some testing and I forgot to remove it ;-)

Reply:
haha, ok. If you don't need it can you also not install it in the task before as well? As well as get rid of the task at the end that stops it.